### ISSUE TITLE
Global transform in Python?

### ISSUE THREAD

**Issue by jfslin** [pyomeca/biorbd#277]
I have a full body kinematic model that I'm using with the Kalman filter and motion capture markers to get IK. Once IK is obtained, I want to rotate the body so that the pelvis of the participant at t=0 is facing perfectly straight, as a way to normalize the orientation of the participant. I thought of a few ways to do this, but ran into issues with each:

1. Rotate the marker data before I feed them into EKF. I saw that `Rotation.fromMarker()` allows me to generate a rotation matrix. I can potentially use this and my ASIS markers to generate the rotation matrix I need to remove the heading offset. I was having some trouble figuring this out though and was wondering if there are any example code? 

2. I can extract the orientation from the pelvis from `model.globalJCS()`. My current code is:

        q_curr = np.array(self.q.data.iloc[0, :])
        jcs = self.model.globalJCS(q_curr, 'Pelvis')
        euler = biorbd.Rotation.toEulerAngles(jcs.rot(), 'xyz').to_array()
        euler[0] = 0
        euler[1] = 0
        rz = biorbd.Rotation(euler, 'xyz')

So now I have this `rz` matrix that I want to rotate all my other segments. I extract the world-to-local position and rotation data to my segments by:

        for i in range(len(self.q)):
            q_curr = self.q.data[i, :]
            timestamp = self.q.time[i]

            x_recons = np.ndarray((self.model.nbSegment() * 6, 1))
            for j, segment in enumerate(self.model.segments()):
                jcs = self.model.globalJCS(q_curr, j)
                i_x = [j * 6, j * 6 + 1, j * 6 + 2]

                x_recons[i_x, 0] = jcs.trans().to_array()
                i_x = [j * 6 + 3, j * 6 + 4, j * 6 + 5]
                x_recons[i_x, 0] = jcs.rot().toEulerAngles(jcs.rot(), 'xyz').to_array()

            self.x_recons.push_index(i, timestamp, x_recons.transpose())

So I figure I should modify `self.model.globalJCS(q_curr, j)` here to `self.model.globalJCS(q_curr, j)*rz'` to get what I need, but it seems like multiplication is not implemented for the `jcs` object, as I get `TypeError: unsupported operand type(s) for *: 'RotoTrans' and 'RotoTrans'`. Is there a way to perform this multiplication? 

3. Lastly, if there is a way to modify the world-to-pelvis transform, I can just modify that to be `rz`. This might even come with the added bonus of updating how `bioviz` looks so I can verify that the rotations were done correctly. However, I wasn't able to find anything like this in the documentations. Is there a world transform I can modify? 

Thank you

**Comment by pariterre** [comment]
Hi there! 
For me the second method is the way to go. I indeed forgot to interface multiply in these matrix. The PR #279 fixes this. It now can be done as such:
```python
import biorbd

a = biorbd.Rotation()
b = biorbd.Rotation()
c = a.multiply(b)  # This performs a @ b
```

If you are unable to update, the workaround is to extract the matrix using to_array, perform the multiplication and then cast back the matrix.
```python
import biorbd
import numpy as np

jcs = biorbd.Rotation_fromEulerAngles(np.array([1, 2, 3]), 'xyz')  # This should be given by jcs
array_jcs = jcs.to_array()
reference = np.random.rand(3, 3)  # This should be any matrix to rotate from
r = reference  @ array_jcs

final = biorbd.Rotation(biorbd.Matrix3d(r[0, 0], r[0, 1], r[0, 2], r[1, 0], r[1, 1], r[1, 2], r[2, 0], r[2, 1], r[2, 2]))
```
Please note this snippet won't work right now as `reference` is not unitary (so the Rotation constructor will return an error)

**Comment by jfslin** [comment]
Thank you for getting this implemented. The latest 'conda-forge' build works. Though `a.multiply(b)` didn't work for me, I 
had to `a*b`, but that is not a problem. 

As a side note, I also had to multiply a vector, to get the rotation applied to the `globalJCS.trans()` part of the segment. The following currently does not work:
```
rot = biorbd.Rotation()
vec = biorbd.Vector3d()
new_vec = rot*vec  # throws TypeError: unsupported operand type(s) for *: 'Rotation' and 'Vector3d'
```

I wrote a function following your suggested workaround (thank you for including that), but saw that `Vector3D` has a `.applyRT` function. So now my current workaround for this while using `biorbd` functions are: 
```
trans = biorbd.Vector3d()
rot_rt = biorbd.RotoTrans_combineRotAndTrans(rot=biorbd.Rotation(), trans=biorbd.Vector3d())
trans.applyRT(rot_rt)
```
I also tried the `RotoTrans` functions and noted that 
```
rt1 = biorbd.RotoTrans()
rt2 = biorbd.RotoTrans()
rt3 = rt1*rt2
```
also throws a `TypeError: unsupported operand type(s) for *: 'RotoTrans' and 'RotoTrans'` error. 

For my purposes, your PR https://github.com/pyomeca/biorbd/pull/279 and my current workaround is sufficient for my needs right now, but just wanted to note this in case anyone is trying to multiply with `Vector3d` and `RotoTrans`.
